examples/hybrid/define_post_processing.jl

```diff
@@ -661,6 +675,516 @@ function paperplots_moist_baro_wave_ρe(sol, output_dir, p, nlat, nlon)
+        ᶜT = @. TD.air_temperature(params, ᶜts)
+        # zonal wind
+        ᶜuₕ = Y.c.uₕ
+        ᶜuₕ_phy = Geometry.UVVector.(ᶜuₕ)
+
+        # assigning to nc obj
+        nc_θ[:, i] = ᶜθ
+        nc_T[:, i] = ᶜT
+        nc_u[:, i] = ᶜuₕ_phy.components.data.:1
+    end
+
+    close(nc)
+
+    ### write out our cubed sphere mesh
+    meshfile_cc = remap_tmpdir * "/mesh_cubedsphere.g"
+    write_exodus(meshfile_cc, hspace.topology)
+
+    meshfile_rll = remap_tmpdir * "/mesh_rll.g"
+    rll_mesh(meshfile_rll; nlat = nlat, nlon = nlon)
+
+    meshfile_overlap = remap_tmpdir * "/mesh_overlap.g"
+    overlap_mesh(meshfile_overlap, meshfile_cc, meshfile_rll)
+
+    weightfile = remap_tmpdir * "/remap_weights.nc"
+    remap_weights(
+        weightfile,
+        meshfile_cc,
+        meshfile_rll,
+        meshfile_overlap;
+        in_type = "cgll",
+        in_np = Nq,
+    )
+
+    ### remap to lat/lon
+    datafile_latlon = output_dir * "/hs-remapped.nc"
+    apply_remap(
+        datafile_latlon,
+        datafile_cc,
+        weightfile,
+        ["PotentialTemperature", "T", "u"],
+    )
+
+    rm(remap_tmpdir, recursive = true)
+
+    ### load remapped data and create statistics for plots
+    datafile_latlon = output_dir * "/hs-remapped" * string(day) * ".nc"
+    ncdata = NCDataset(datafile_latlon, "r")
+    lat = ncdata["lat"][:]
+    z = ncdata["z"][:]
+    time = ncdata["time"][:]
+
+    T = ncdata["T"][:, :, :, time .> 3600 * 24 * 200]
+    θ = ncdata["PotentialTemperature"][:, :, :, time .> 3600 * 24 * 200]
+    u = ncdata["u"][:, :, :, time .> 3600 * 24 * 200]
+
+    u_timeave_zonalave = calc_zonalave_timeave(u)
+    T_timeave_zonalave = calc_zonalave_timeave(T)
+    θ_timeave_zonalave = calc_zonalave_timeave(θ)
+
+    T2_timeave_zonalave = calc_zonalave_variance(T)
+
+    Plots.png(
+        Plots.contourf(
+            lat,
+            z,
+            u_timeave_zonalave',
+            color = :balance,
+            clim = (-30, 30),
+            linewidth = 0,
+            yaxis = :log,
+            title = "u",
+            xlabel = "lat (deg N)",
+            ylabel = "z (m)",
+        ),
+        output_dir * "/hs-u.png",
+    )
+
+    Plots.png(
+        Plots.contourf(
+            lat,
+            z,
+            T_timeave_zonalave',
+            levels = 190:10:310,
+            clim = (190, 310),
+            contour_labels = true,
+            yaxis = :log,
+            title = "u",
+            xlabel = "lat (deg N)",
+            ylabel = "z (m)",
+        ),
+        output_dir * "/hs-T.png",
+    )
+
+    Plots.png(
+        Plots.contourf(
+            lat,
+            z,
+            θ_timeave_zonalave',
+            levels = 260:10:360,
+            clim = (260, 360),
+            contour_labels = true,
+            yaxis = :log,
+            title = "u",
+            xlabel = "lat (deg N)",
+            ylabel = "z (m)",
+        ),
+        output_dir * "/hs-θ.png",
```

Comment:
```suggestion
        output_dir * "/hs-theta.png",
```
(I just prefer no unicode in filenames, but that's up to you)